todo_app/data/session_items.py

```diff
@@ -1,22 +1,41 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+from todo_app.data.item import Item
+import requests
+import os
 
 def custom_sort(t):
     return t['status']
 
+def get_lists():
+    """
+    Gets all lists on a board 
+    """
+    params = {
+        'key': os.environ['ACCOUNT_KEY'],
+        'token': os.environ['SECRET_KEY']
```

Comment:
Here it looks like you're overloading the secret key as both the encryption key for the session (which is what SECRET_KEY was doing previously) and now using it to store the Trello token - keeping those two separate would make more sense

Reply:
this has been done - i didnt realize that the `SECRET_KEY` was reserved. These have been renamed to `TRELLO_ACCOUNT_KEY` and `TRELLO_SECRET_KEY`